docs/examples/qaoa_qiskit.ipynb

```diff
@@ -0,0 +1,713 @@
+    "As individual terms in the summation of $H_P$ and $H_B$ both commute, we can write the unitaries as:\n",
+    "\n",
+    "$$ U(H_B) = e^{-i \\beta H_B} = e^{-i \\beta X_0}e^{-i \\beta X_1}e^{-i \\beta X_2}e^{-i \\beta X_3}.$$\n",
+    "\n",
+    "Notice that each term in the product above corresponds to an X-rotation on each qubit. And we can write $U(H_P)$ as:\n",
+    "\n",
+    "$$ U(H_P) = e^{-i \\gamma H_P} = e^{-i \\gamma Z_0 Z_1}e^{-i \\gamma Z_1 Z_2}e^{-i \\gamma Z_2 Z_3}e^{-i \\gamma Z_0 Z_3}$$\n",
+    "\n",
+    "Let's now examine what the circuits of the two unitaries look like. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### The Mixing Unitary"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/html": [
+       "<pre style=\"word-wrap: normal;white-space: pre;background: #fff0;line-height: 1.1;font-family: &quot;Courier New&quot;,Courier,monospace\">     ┌───────────────┐\n",
+       "q_0: ┤ RX(2*$\\beta$) ├\n",
+       "     ├───────────────┤\n",
+       "q_1: ┤ RX(2*$\\beta$) ├\n",
+       "     ├───────────────┤\n",
+       "q_2: ┤ RX(2*$\\beta$) ├\n",
+       "     ├───────────────┤\n",
+       "q_3: ┤ RX(2*$\\beta$) ├\n",
+       "     └───────────────┘</pre>"
+      ],
+      "text/plain": [
+       "     ┌───────────────┐\n",
+       "q_0: ┤ RX(2*$\\beta$) ├\n",
+       "     ├───────────────┤\n",
+       "q_1: ┤ RX(2*$\\beta$) ├\n",
+       "     ├───────────────┤\n",
+       "q_2: ┤ RX(2*$\\beta$) ├\n",
+       "     ├───────────────┤\n",
+       "q_3: ┤ RX(2*$\\beta$) ├\n",
+       "     └───────────────┘"
+      ]
+     },
+     "execution_count": 3,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "from qiskit import QuantumCircuit, ClassicalRegister, QuantumRegister\n",
+    "from qiskit import Aer, execute\n",
+    "from qiskit.circuit import Parameter\n",
+    "\n",
+    "# Adjacency is essentially a matrix which tells you which nodes are\n",
+    "# connected. This matrix is given as a sparse matrix, so we need to\n",
+    "# convert it to a dense matrix\n",
+    "adjacency = nx.adjacency_matrix(G).todense()\n",
+    "\n",
+    "nqubits = 4\n",
+    "\n",
+    "beta = Parameter(\"$\\\\beta$\")\n",
+    "qc_mix = QuantumCircuit(nqubits)\n",
+    "for i in range(0, nqubits):\n",
+    "    qc_mix.rx(2 * beta, i)\n",
+    "    \n",
+    "qc_mix.draw()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### The Problem Unitary"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/html": [
+       "<pre style=\"word-wrap: normal;white-space: pre;background: #fff0;line-height: 1.1;font-family: &quot;Courier New&quot;,Courier,monospace\">                                  ░                              ░      »\n",
```

Comment:
This graph looks much nicer in the Qiskit docs than in ours, is there a plugin or something we can use to render the symbols like they do?

### Ours
<img width="816" alt="Screen Shot 2021-07-21 at 8 37 06 AM" src="https://user-images.githubusercontent.com/43723790/126507508-d076576c-0b21-4800-bbcb-882ff0d38115.png">

### Theirs
<img width="679" alt="Screen Shot 2021-07-21 at 8 39 19 AM" src="https://user-images.githubusercontent.com/43723790/126507562-f15309d1-3a44-4f15-84e1-28df7ade5f6c.png">

Reply:
I noticed that. It's a nice to have, but not a priority at the moment